### PR TITLE
Ohri 2124-Added tptTreatment tab on the program management link board.

### PR DESCRIPTION
**Pull request by kirwea**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes the ticket number in the format `OHRI-123 My PR title`.
- [ ] My work includes tests or is validated by existing tests.

## Screenshots
Screenshot from 2024-02-06 17-31-13

### CODE REVIEW

File: packages/esm-tb-app/src/views/tpt/program-management/tabs/tpt-treatment.component.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            intent: '*',
+            label: t('editForm', 'Edit Form'),
+            mode: 'edit',
+          },
+        ],
+      },
+    ],
+    [],
+  );
+  // eslint-disable-next-line no-empty-pattern
```
Comment (ODORA0) [inline packages/esm-tb-app/src/views/tpt/program-management/tabs/tpt-treatment.component.tsx:67]: What is the purpose of this?

File: packages/esm-tb-app/src/views/tpt/program-management/tabs/tpt-treatment.component.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      encounterType={encounterTypes.tptTreatmentAndFollowUp}
+      formList={[{ name: formNames.TptTreatmentFormName }]}
+      description={headerTitle}
+      headerTitle={headerTitle}
+      columns={columns}
+      launchOptions={{
+        moduleName: '',
+        hideFormLauncher: false,
+        displayText: t('add', 'Add'),
+        workspaceWindowSize: 'minimized',
```
Comment (ODORA0) [inline packages/esm-tb-app/src/views/tpt/program-management/tabs/tpt-treatment.component.tsx:81]: You need to remove this line too

**Review by ODORA0 — CHANGES_REQUESTED** [review]

File: packages/esm-tb-app/src/views/tpt/program-management/tabs/tpt-treatment.component.tsx (hunk 2 above)
Comment (kirwea) [reply packages/esm-tb-app/src/views/tpt/program-management/tabs/tpt-treatment.component.tsx:81]: i have made the necessary changes.

**Review by ODORA0 — APPROVED** [review]

**Review by arodidev — APPROVED** [review]

**Comment by arodidev** [comment]
LGTM @kirwea

File: packages/esm-tb-app/src/views/tpt/program-management/tabs/tpt-treatment.component.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  const { obsConcepts, encounterTypes, formNames } = useConfig();
+  const headerTitle = t('tptTreatment', 'TPT Treatment');
+  const columns: EncounterListColumn[] = useMemo(
+    () => [
+      {
+        key: 'caseId',
+        header: t('caseId', 'Case Id'),
+        getValue: (encounter: any) => {
+          return getObsFromEncounter(encounter, obsConcepts.caseID);
+        },
```
Comment (pirupius) [inline packages/esm-tb-app/src/views/tpt/program-management/tabs/tpt-treatment.component.tsx:22]: I was of the understanding that the treatment form doesn't have this value. Has this changed?

**Review by pirupius — CHANGES_REQUESTED** [review]

File: packages/esm-tb-app/src/views/tpt/program-management/tabs/tpt-treatment.component.tsx (hunk 3 above)
Comment (kirwea) [reply packages/esm-tb-app/src/views/tpt/program-management/tabs/tpt-treatment.component.tsx:22]: it does have that value,i confirmed with Veronica

**Review by pirupius — APPROVED** [review]

**Pull request merged**